EcpBtBot/MainClass.py

```diff
@@ -1,39 +1,42 @@
 
 import os
+import logging
 from .DialogFlow.API import *
 from .DialogFlow.DialogFlowResponse import *
 from .Conversational_Integration import *
 
-print("Dire Bonjour pour commencer \n")
 
 dico_users = {}
 #dictionnaire (var globale): pour chaque clé (user), on a un dictionnaire de connexions API en fonction des thèmes
 
 class API_reponse:
 
     def __init__(self, sujet):
-        if sujet == "sepresenter":
+        if sujet == "Se_presenter":
             self.ai = API_SePresenter()
-        elif sujet == "hotel":
+        elif sujet == "Hôtel":
             self.ai = API_Hotel()
-        elif sujet == "restaurant":
+        elif sujet == "Restaurant":
             self.ai = API_Restaurant()
-        elif sujet == "boulangerie":
+        elif sujet == "Boulangerie":
             self.ai = API_Boulangerie()
+        elif sujet == "intro":
+            self.ai = API_Intro()
+        else: 
+            logging.critical("Sujet non defini")
 
     def reponseBot(self, message):
         #user_id = user
         user_message = message
         rep = DialogFlowResponse(self.ai._get_json_response(user_message))
+        
         rep_and_qr=Reponse_And_QuickReplies(rep)
 
         return rep_and_qr
     
 class Reponse_And_QuickReplies:
```

Comment:
Better remove the underscores and make this class inherit from object. it's cleaner.